diff_test.go

```diff
@@ -752,6 +752,51 @@ func TestSchemaDiffRoutines(t *testing.T) {
 	}
 }
 
+func TestSchemaDiffFilteredViewDiffs(t *testing.T) {
+	s1v1 := aView()
+	s1 := aSchemaWithViews("s1", &s1v1)
+
+	s2v1 := anotherView()
+	s2v2 := anAlterView()
+
+	s2 := aSchemaWithViews("s1", &s2v1, &s2v2)
+
+	assertFiltered := func(sd *SchemaDiff, expectLen int, types ...DiffType) {
+		t.Helper()
```

Comment:
I'm not sure this will have the expected behavior unless you pass `t` explicitly as a closure parameter. Would recommend doing so to be safe.